src/trans/internal/leinv_mod.F90

```diff
@@ -138,19 +130,7 @@ SUBROUTINE LEINV(KM,KMLOC,KFC,KIFC,KF_OUT_LT,KSL,KDGLU,PIA,PAOA1,PSOA1)
   IF(ILA <= ITHRESHOLD .OR. .NOT.S%LUSEFLT) THEN
 
     IF (LHOOK) CALL DR_HOOK('LEINV_'//CLX//'GEMM_1',0,ZHOOK_HANDLE)
-    IF (LLDOUBLE) THEN
-      CALL DGEMM('N','N',KDGLU,KIFC,ILA,1.0_JPRD,S%FA(KMLOC)%RPNMA,KDGLU,&
-       &ZBA,ILA,0._JPRD,ZC,KDGLU)
-    ELSE
-       IF (LL_IEEE_HALT) THEN
-          call ieee_get_halting_mode(ieee_invalid,LL_HALT_INVALID)
-          if (LL_HALT_INVALID) call ieee_set_halting_mode(ieee_invalid,.false.)
-       ENDIF
-       CALL SGEMM('N','N',KDGLU,KIFC,ILA,1.0_JPRM,S%FA(KMLOC)%RPNMA,KDGLU,&
-            &ZBA,ILA,0._JPRM,ZC,KDGLU)
-       if (LL_IEEE_HALT .and. LL_HALT_INVALID) call ieee_set_halting_mode(ieee_invalid,.true.)
-    ENDIF
-    IF (LHOOK) CALL DR_HOOK('LEINV_'//CLX//'GEMM_1',1,ZHOOK_HANDLE)
```

Comment:
I think you need to keep this closing Dr Hook call.

Reply:
This is why an extra pair of eyes is so useful!!! Thanks!

Reply:
Fixed now.